src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -593,6 +593,17 @@ trait EventFormats {
       "runSpecId" -> change.runSpecId,
       "agentId" -> change.instance.agentInfo.agentId,
       "host" -> change.instance.agentInfo.host,
+      "version" -> change.version,
```

Comment:
The version originates from the runSpec that started this instance.
I would agree that makes sense.